daemon/Makefile

```diff
@@ -0,0 +1,85 @@
+#
+# Build the daemon and related utilities
+#
+
+include ../make/version.mk
+-include ../make/secrets.mk
+
+#
+# Global variables
+#
+
+# Executable targets
+BINARIES := client daemon integration_runner stressor
+
+# Default go invocation command
+GO := go
+
+# module path
+GO_MODULE := github.com/newrelic/newrelic-php-agent/daemon
+
+# Pass relevant flags from secrets.mk && version.mk to the version and secrets packages
+GOFLAGS := -ldflags '-X $(GO_MODULE)/internal/newrelic/version.Number=$(AGENT_VERSION) 							\
+					-X $(GO_MODULE)/internal/newrelic/version.Commit=$(GIT_COMMIT) 								\
```

Comment:
This looks like it needs formatting but I'm having trouble making a suggestion via GH.

Reply:
This is due to tabs being differently aligned in my editor vs GH. Try checking out the PR and viewing in your editor- if it looks bad in the editor, we can change it. But if it's just GH, we're probably better off leaving it. In my editor this is all aligned.